csharp/Helper/DataBaseHelper.cs

```diff
@@ -0,0 +1,16 @@
+using Cassandra;
+
+namespace MediaPlayer.Helper;
+
+public static class DataBaseHelper
+{
+    public static bool RowSetHasResult(RowSet rowSet)
+    {
+        var rows = rowSet
+            .GetRows()
+            .ToList();
+
+        var hasResult = rows.Count > 0;
+        return hasResult;
```

Comment:
There's two better options IHMO, instead allocate a new variable to return the expression.

```suggestion
        var rows = rowSet
            .GetRows()
            .ToList();

        return rows.Count > 0;
```

```suggestion
        return rowSet
            .GetRows()
            .ToList()
            .Count > 0;
```